index.html

```diff
@@ -0,0 +1,165 @@
+          facere quae, dolore perspiciatis iste sapiente ut a animi! Voluptate
+        </p>
+      </div>
+      <div id="time">
+        <h2>2020.10.15(THU) ~ 16(FRI)</h2>
+        <h6>@National Museum of Rwanda, Art Center</h6>
+      </div>
+    </div>
+
+    <section class="main-program">
+      <h2>Main Programs</h2>
+              <hr>
+      <div class="program-list">
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_01.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Lecture</h3>
+          </div>
+          <div class="p-paragraph">
+            <p>Appreciate  various creations applying CC license of artists, organized from Art Center Nabi.</p>
+          </div>
+        </div>
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_01.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Lecture</h3>
+          </div>
+          <div class="p-paragraph">
+            <p>Appreciate  various creations applying CC license of artists, organized from Art Center Nabi.</p>
+          </div>
+        </div>
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_01.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Lecture</h3>
+          </div>
+          <div class="p-paragraph">
+            <p>Have time to share your thoughts and opinions with experts for each topic.</p>
+          </div>
+        </div>
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_04.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Lecture</h3>
+          </div>
+          <div class="p-paragraph">
+            <p>Try creating your own work using open source license rather than just watching.</p>
+          </div>
+        </div>
+        <div class="programs">
+          <div class="p-icon"><a href="#"><img src="./Images/program_icon_02.png" alt=""></a></div>
+          <div class="p-header">
+            <h3>Legend</h3>
+          </div>
+          <div class="p-paragraph  p-space">
+            <p>Get opportunities to work wit CC affiliates around the world, also after the summit.</p>
+          </div>
+        </div>
+      </div>
+      <div class="see-the-program">
+        <a href="#">Checkout all the programs</a>
+      </div>
+    </section>
+    <div id="featured-speakers">
+      <h2>Featured Speakers</h2>
+      <hr>
+    </div>
+    <section id="speakers" >
+        
+    </section>
+
+    <footer>
+      <div id="footer-div">
+        <h2>Partners</h2>
+         <hr>
+         <div class="footer-images">
+         <img src="./Images/apple-logo'.png" alt="">
+         <img src="./Images/android-logo-0.png" alt="">
+         <img src="./Images/airbnb.png" alt="">
+        </div>
+    </footer>
```

Comment:
- [ ] As requested by the previous reviewer, After updating the partner section, let's add the missing footer to our webpage. The home and about pages should have the same footer.

- Kindly pass close attention to the layout alignment, background color, color, and font size of the footer section.
- This change is applicable to both mobile and desktop versions of your webpage
![image](https://user-images.githubusercontent.com/92303891/204086617-0b33c001-a1bd-4d70-b181-f6042b4f8910.png)

- [ ] Please on the desktop screen the background color of the `partners` section does not match the design.
![Screenshot from 2022-11-26 11-35-34](https://user-images.githubusercontent.com/92303891/204086694-f5bf2766-02e8-4143-bec4-5d66bb3a06b8.png)

Reply:
Thanks! @zieeco